src/Sylius/Bundle/WebBundle/Resources/views/Backend/Order/_shippingState.html.twig

```diff
@@ -13,4 +13,4 @@
     {% set label = 'info' %}
 {% endif %}
 
-<span class="label label-{{ label }}">{{ state|title }}</span>
+<span class="label label-{{ label }}">{{ ('sylius.account.order.shipment.' ~ shipment.state) |trans({'%at%': shipment.updatedAt|date('d.m.Y h:i')}) }}</span>
```

Comment:
You should not hardcode the date format, pershaps you should use `localizateddate` instead.